examples/etcd/etcd-in-cluster/README.md

```diff
@@ -0,0 +1,350 @@
+deployment.apps/nginx created
+
+» kubectl get all -n nginx
+NAME                         READY   STATUS    RESTARTS   AGE
+pod/nginx-86c57db685-ztb7l   1/1     Running   0          37s
+
+NAME                    READY   UP-TO-DATE   AVAILABLE   AGE
+deployment.apps/nginx   1/1     1            1           40s
+
+NAME                               DESIRED   CURRENT   READY   AGE
+replicaset.apps/nginx-86c57db685   1         1         1       40s
+```
+
+## Protect the Application
+
+We can now take snapshot of the ETCD server that is running by creating backup actionset that is going to execute backup phase from the blueprint that we have
+created above
+
+**Note**
+
+Pleae make sure to change the **profile name**, the **ETCD pod name**, **pod namespace** and **blueprint name** in the `backup-actionset.yaml` manifest file.
+
+```
+# find the profile name
+» kubectl get profile -n kanister
+NAME               AGE
+s3-profile-nnvmm   54s
+
+# find blueprint name
+» kubectl get blueprint -n kanister
+NAME             AGE
+etcd-blueprint   41s
+
+# create actionset
+» kubectl create -f backup-actionset.yaml -n kanister
+actionset.cr.kanister.io/backup-hnp95 created
+
+# you can check the status of the actionset by describing it and making sure that it is succeeded
+» kubectl describe actionsets.cr.kanister.io -n kanister backup-hnp95
+
+# You should see events like this
+...
+...
+Events:
+  Type    Reason           Age   From                 Message
+  ----    ------           ----  ----                 -------
+  Normal  Started Action   34s   Kanister Controller  Executing action backup
+  Normal  Started Phase    34s   Kanister Controller  Executing phase takeSnapshot
+  Normal  Ended Phase      34s   Kanister Controller  Completed phase takeSnapshot
+  Normal  Started Phase    34s   Kanister Controller  Executing phase uploadSnapshot
+  Normal  Ended Phase      0s    Kanister Controller  Completed phase uploadSnapshot
+  Normal  Started Phase    0s    Kanister Controller  Executing phase removeSnapshot
+  Normal  Ended Phase      0s    Kanister Controller  Completed phase removeSnapshot
+  Normal  Update Complete  0s    Kanister Controller  Updated ActionSet 'backup-hnp95' Status->complete
+```
+
+Once the backup actionset is complete, we can check the object storage to make the backup is uploaded successfully.
+
+## Imitate Disaster
+
+Now let's assume something went wrong and we lost the test namespace that we created eariler. Please delete the namespace manually
+
+```
+» kubectl delete ns nginx
+namespace "nginx" deleted
+
+» kubectl get all -n nginx
+No resources found in nginx namespace.
+```
+
+## Restore the ETCD cluster
+
+To restore the ETCD cluster you will have to have the backup location, where the backup action uploaded the snapshot so that you can download the snapshot. To figure out
+the backup location describe the actionset to check the output of the `uploadSnapshot` phase.
```

Comment:
If we store the value in env var, we can read backuppath from env var in the next commands. So user can just copy paste those commands

Reply:
I have added the kubectl command but storing in env var didnt look good because we are going to execute the next command into one of the leader hosts. So user will anyway have to copy the backup location.